internal/render/table.go

```diff
@@ -3,28 +3,44 @@ package render
 import (
 	"io"
 
-	"github.com/olekukonko/tablewriter"
+	"github.com/jedib0t/go-pretty/v6/table"
+)
+
+const space = " "
+
+var (
+	tableStyle = table.Style{
+		Name: "StyleDefault",
+		Box: table.BoxStyle{
+			EmptySeparator: space,
+			PaddingLeft:    "", // empty to avoid white symbol at the beginning of every line
+			PaddingRight:   space + space + space,
+		},
+		Color:   table.ColorOptionsDefault,
+		Format:  table.FormatOptionsDefault,
+		HTML:    table.DefaultHTMLOptions,
+		Options: table.OptionsNoBordersAndSeparators,
+		Size:    table.SizeOptionsDefault,
+		Title:   table.TitleOptionsDefault,
+	}
 )
 
 // Table renders the table with the provided headers and data
-func Table(writer io.Writer, modulesData [][]string, headers []string) {
-	twTable := setTable(writer)
-	twTable.AppendBulk(modulesData)
-	twTable.SetHeader(headers)
-	twTable.Render()
+func Table(writer io.Writer, headers []interface{}, rows [][]interface{}) {
```

Comment:
Why do you use `interface{}` instead of `string`?

Reply:
because new library requires such input data types

Reply:
ok